standard-covoiturage_openapi.yaml

```diff
@@ -504,3 +651,96 @@ components:
                 provider mobile app on the screen presenting the specific journey.
+                                
+    ConnectionMessage:
+      type: object
+      required:
+        - driverJourneyId
+        - driverUserid
+        - passengerJourneyId
+        - passengerUserid
+      properties:
+        driverJourneyId:
+          type: string
+          format: uuid
+          description: |-
+            Unique journey's id in the format of an [UUID](https://datatracker.ietf.org/doc/html/rfc4122).
+            Usage of a [4 UUID](https://datatracker.ietf.org/doc/html/rfc4122#section-4.4) generation algorithm
+            is advised.
+        driverUserid:
+          type: string
+          description: User's id as retrieved by the route GET /driver_journeys.
+        driverAlias:
+          type: string
+          description: Driver's name alias.
+        details:
+          type: string
+          maxLength: 500
+          description: Free text content of the message. The message can contain all the details (phone number, email, etc.) allowing the recipient to call back the sender in order to carpool with him/her.
+        passengerJourneyId:
+          type: string
+          format: uuid
+          description: |-
+            Unique journey's id in the format of an [UUID](https://datatracker.ietf.org/doc/html/rfc4122).
+            Usage of a [4 UUID](https://datatracker.ietf.org/doc/html/rfc4122#section-4.4) generation algorithm
+            is advised.
+        passengerUserid:
+          type: string
+          description: User's id as retrieved by the route GET /passenger_journeys.
+        passengerAlias:
+          type: string
+          description: Passenger's name alias.
+        driverOperator:
+          $ref: '#/components/schemas/operator'
+        driverOperatorWebUrl:
+          $ref: '#/components/schemas/operatorUrl'
+        passengerOperator:
+          $ref: '#/components/schemas/operator'
+        passengerOperatorWebUrl:
+          $ref: '#/components/schemas/operatorUrl'
+          
+    Booking:
+      allOf:
+        - $ref: '#/components/schemas/ConnectionMessage'
+      type: object
+      required:
+        - id
+        - frequency
+        - outwardSchedule
+        - price
+      properties:
+        id:
+          $ref: '#/components/schemas/bookingId'
+        passengerPickupDate:
+          type: number
+          description: |-
+            Passenger pickup datetime as a UNIX UTC timestamp in seconds.
+          format: long
+        status:
+          $ref: '#/components/schemas/bookingStatus'
+        price:
+          type: number
+          description: Proposed and agreed (when `status=accepted`) price for the journey.
+      
+    bookingId:
+      type: string
+      description: Booking id is common between both operators, and must be set as a MD5 hashing of a string concatenating the `driverJourneyId`, the `passengerJourneyId` and a UNIX UTC timestamp in seconds.
```

Comment:
I comment myself : I was not in favor of UUID for the journeyID as explained here https://github.com/fabmob/standard-covoiturage/pull/2#issuecomment-1074873324, but it could be an option for the booking ID as it is a new ID and has no impact on the operators existing infrastructures

Reply:
I agree. A UUID would be even more interesting here as it would prevent collisions while generated by multiple operators.